pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <properties>
         <java.version>17</java.version>
-        <starter.version>1.20.1</starter.version>
+        <starter.version>1.22.2</starter.version>
```

Comment:
bump with the [latest](https://github.com/pagopa/pagopa-api-config-starter/releases) 1.23.0